rtp_h264_extractor.lua

```diff
@@ -184,19 +188,20 @@ do
             local payloadTable = { h264_data() }
             local seqTable = { rtp_seq() }
             
-            if (#payloadTable) ~= (#seqTable) then 
-                log("ERROR: payloadTable size is "..tostring(#payloadTable)..", seqTable size is "..tostring(#seqTable))
-                return
-            end
+--            if (#payloadTable) ~= (#seqTable) then 
+--                log("ERROR: payloadTable size is "..tostring(#payloadTable)..", seqTable size is "..tostring(#seqTable))
+--                return
+--            end
```

Comment:
To handle that case i suggest: may be we should check #payloadTable >= #seqTable else if #payloadTable < #seqTable, then your error?
Added code at my fork.